controllers/storagecluster/rbd_mirror_debug.go

```diff
@@ -0,0 +1,129 @@
+// This is a temporary measure to enable verbose logging for RDR
+// This should be removed once RDR goes to GA.
+package storagecluster
+
+import (
+	"context"
+	"os"
+
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+func (r *StorageClusterReconciler) ensureRbdMirrorDebug(sc *ocsv1.StorageCluster) error {
+	if sc.Spec.Mirroring.Enabled {
+		return r.Client.Create(context.TODO(), rbdMirrorDebugJob(sc))
+	}
+	return nil
+}
+
+func rbdMirrorDebugJob(sc *ocsv1.StorageCluster) *batchv1.Job {
+	rookImage := os.Getenv("ROOK_CEPH_IMAGE")
+	debugJob := &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rbd-mirror-debug",
+			Namespace: sc.Namespace,
```

Comment:
Add ownerRef for this job to get deleted when rook/ocs is uninstalled.

Reply:
Thanks for pointing it out, Will do it.